model/src/main/proto/spine_examples/shareaware/market/commands.proto

```diff
@@ -52,3 +52,19 @@ message ObtainShares {
     // The quantity of the shares that the purchase process wants to obtain.
     int32 quantity = 4 [(required) = true];
 }
+
+// A command to realize shares on the market.
```

Comment:
I don't understand what you mean here. And the documentation does not help. The problem is with "realize" verb, which is repeated, and which to my mind, does not fit.

Please come up with a better name and/or more detailed explanation (so that I could come up with an alternative name).